tests/unit/aiplatform/test_lro.py

```diff
@@ -26,6 +30,18 @@
 
 
 TEST_OPERATION_NAME = "test/operation"
+_TEST_PROJECT = "test-project"
+
+
+def setup_module(module):
```

Comment:
is this used?

Reply:
https://docs.pytest.org/en/stable/xunit_setup.html#module-level-setup-teardown

Why don't you think so?

Reply:
I see - thanks.  I have not used this before.